packages/manager/modules/pci/src/projects/project/instances/instance/backup/backup.controller.js

```diff
@@ -75,17 +75,15 @@ export default class PciInstanceBackupController {
 
   formatPrice() {
     const { price } = this.monthlyPrice;
-    if (this.instance.isLocalZone) {
-      const { value, currencyCode } = price;
-      return `~${new Intl.NumberFormat(
-        this.coreConfig.getUserLocale().replace('_', '-'),
-        {
-          style: 'currency',
-          currency: currencyCode,
-          maximumFractionDigits: 3,
-        },
-      ).format(value * HOURS_PER_MONTH)}`;
-    }
-    return price.text;
+
+    const { value, currencyCode } = price;
+    return `~${new Intl.NumberFormat(
```

Comment:
Be sure to put the number format outside for better performances, only the `format` call needs to be inside this function